featurehub-javascript-client-sdk/app/middleware.ts

```diff
@@ -143,6 +144,17 @@ class BaggageHolder implements FeatureStateHolder {
     return this.getType();
   }
 
+  get value(): T {
+    const v = {
+      [FeatureValueType.Boolean]: this.getBoolean(),
+      [FeatureValueType.String]: this.getString(),
+      [FeatureValueType.Number]: this.getNumber(),
+      [FeatureValueType.Json]: this.getRawJson() ?? {},
```

Comment:
getRawJson() is always the raw text, if I were you I would probably remove the parseJson parameter and then do that here in this function?